app/javascript/components/layout/CustomTable.js

```diff
@@ -235,35 +235,39 @@ class CustomTable extends React.Component {
               </Col>
               <span className="ml-2 text-nowrap align-self-center">{`Displaying ${this.props.rowData.length} out of ${this.props.totalRows} rows.`}</span>
             </Row>
-          </Form>
-          {this.props.totalRows > 0 && (
-            <ReactPaginate
-              className=""
-              disableInitialCallback={true}
-              pageCount={Math.ceil(this.props.totalRows / this.props.entries)}
-              pageRangeDisplayed={4}
-              marginPagesDisplayed={1}
-              initialPage={this.props.page}
-              forcePage={this.props.page}
-              onPageChange={this.props.handlePageUpdate}
-              previousLabel="Previous"
-              nextLabel="Next"
-              breakLabel="..."
-              containerClassName="pagination mb-0"
-              activeClassName="active"
-              disabledClassName="disabled"
-              pageClassName="paginate_button page-item"
-              previousClassName="paginate_button page-item"
-              nextClassName="paginate_button page-item"
-              breakClassName="paginate_button page-item"
-              pageLinkClassName="page-link text-primary"
-              previousLinkClassName={this.props.page === 0 ? 'page-link' : 'page-link text-primary'}
-              nextLinkClassName={this.props.page === Math.ceil(this.props.totalRows / this.props.entries) - 1 ? 'page-link' : 'page-link text-primary'}
-              activeLinkClassName="page-link text-light"
-              breakLinkClassName="page-link text-primary"
-            />
-          )}
-        </div>
+          </Col>
+          <Col xs="24" className="mt-2 custom-table-footer-column-two">
+            {this.props.totalRows > 0 && (
+              <div style={{ float: 'right' }}>
+                <ReactPaginate
+                  className=""
+                  disableInitialCallback={true}
+                  pageCount={Math.ceil(this.props.totalRows / this.props.entries)}
+                  pageRangeDisplayed={4}
```

Comment:
Another possible thought to give us more room at small screens... could we shrink this value when certain breakpoints are hit so less pages are shown in the pagination?

Reply:
Not 100% sure how I'd implement this, but it's definitely worth looking into! I'll take a look later today or tomorrow when I get some time.

Reply:
I also like this idea